ATTIC/logsupport.cpp

```diff
@@ -17,15 +17,19 @@
 #include "logsupport.hpp"
 #include "../srtcore/srt.h"
 #include "../srtcore/utilities.h"
+#include "ofmt.h"
```

Comment:
NOTE: this and some other files were moved to `ATTIC` so that they can stay as reference just in case, but they can be just as well removed. Changes visible here are only intermediate attempts and abandoned.